experiment/posttest.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "question": "1.How many pass transistors are required to build 2 X 1 mux circuit?",
+        "answers": {
+            "a": "2 ",
+            "b": "4 ",
+            "c": "0 ",
+            "d": "1 "
+        },
+        "correctAnswer": "a"
+    },
+    {
+        "question": "2. For negative edge pass tansistor the clock should be connected to?",
+        "answers": {
+            "a": "Drain of pmos",
+            "b": "Source of nmos",
+            "c": "Gate of nmos",
+            "d": "Gate of pmos"
+        },
+        "correctAnswer": "d"
+    },
+    {
+        "question": "3. Control logic in the mux circuit is given by",
+        "answers": {
+            "a": "Clock",
+            "b": "ClockComplement",
+            "c": "DataLine1",
+            "d": "DataLine2"
+        },
+        "correctAnswer": "b"
+    },
+    {
+        "question": "4. Which are following statements are true?",
+        "answers": {
+            "a": "Latches are used as temporary buffers  ",
```

Comment:
Please remove extra spaces at the end of the answer

Reply:
Removed